topdown/parse_bytes_test.go

```diff
@@ -24,6 +24,9 @@ func TestNumBytes(t *testing.T) {
 			{"10 KiB mixed case", `10Kib`, 10 * ki},
 			{"200 megabytes as mb", `200mb`, 200 * mb},
 			{"300 GiB", `300GiB`, 300 * gi},
+			{"100 megabytes as m", `100m`, 100 * mb},
+			{"100 gibibytes as gi", `100gi`, 100 * gi},
+			{"20 terrabyte as t", `20t`, 20 * tb},
```

Comment:
As long as we're in here, can you add a test case for each of the units? Its kind of tedious but having a test case for eg `k`, `ki`, `kb`, `kib`, etc etc would be nice for catching regressions/typos in the implementation.

Reply:
Should have every case covered now